sdk/textanalytics/ai-text-analytics/src/generated/models/parameters.ts

```diff
@@ -1,48 +1,46 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import * as coreHttp from "@azure/core-http";
+import * as Mappers from "../models/mappers";
 
 export const endpoint: coreHttp.OperationURLParameter = {
-  parameterPath: "endpoint",
+  parameterPath: "Endpoint",
   mapper: {
-    required: true,
+    type: { name: "String" },
     serializedName: "Endpoint",
-    defaultValue: '',
-    type: {
-      name: "String"
-    }
+    required: true
   },
   skipEncoding: true
 };
-export const includeStatistics: coreHttp.OperationQueryParameter = {
-  parameterPath: [
-    "options",
-    "includeStatistics"
-  ],
+
+export const modelVersion: coreHttp.OperationQueryParameter = {
+  parameterPath: ["options", "model-version"],
   mapper: {
-    serializedName: "showStats",
-    type: {
-      name: "Boolean"
-    }
+    type: { name: "String" },
+    serializedName: "model-version"
   }
 };
-export const modelVersion: coreHttp.OperationQueryParameter = {
-  parameterPath: [
-    "options",
-    "modelVersion"
-  ],
+
+export const showStats: coreHttp.OperationQueryParameter = {
```

Comment:
This needs to be renamed to includeStatistics. I'm following up with Autorest.Modelerfour to find out why it was not renamed by the transform in the README.md

Azure/autorest.modelerfour/issues/145